CMakeLists.txt

```
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.19)
```

```suggestion
cmake_minimum_required(VERSION 3.16...3.19)
```

3.16 is the latest version of CMake available on focal.